app/src/main/java/edu/uw/tcss450/chatapp/ui/auth/signin/ChangePassword.java

```diff
@@ -0,0 +1,24 @@
+package edu.uw.tcss450.chatapp.ui.auth.signin;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import edu.uw.tcss450.chatapp.R;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class ChangePassword extends Fragment {
```

Comment:
Im not crazy about the name. I think PasswordVerification would be more appropriate

Reply:
Also you should add the word Fragment to the name, so ChangePasswordFragment. I